engineio/base/conn_test.go

```diff
@@ -37,45 +38,52 @@ func TestOpError(t *testing.T) {
 		temporary bool
 		errString string
 	}{
-		{"http://domain/abc", "post(write) to", io.EOF, false, false, "post(write) to http://domain/abc: EOF"},
-		{"http://domain/abc", "get(read) from", io.EOF, false, false, "get(read) from http://domain/abc: EOF"},
-		{"http://domain/abc", "post(write) to", fakeOpError{true, false}, true, false, "post(write) to http://domain/abc: fake error"},
-		{"http://domain/abc", "get(read) from", fakeOpError{false, true}, false, true, "get(read) from http://domain/abc: fake error"},
+		{"http://domain/abc", "post(write) to", io.EOF, false, false,
```

Comment:
If wrapping test cases, please cut them in the same column. I prefer to cut at `timeout` field.

Reply:
Sorry, i don't understand. What are you suggest?

Reply:
```
{"http://domain/abc", "post(write) to", io.EOF,
	false, false, "post(write) to http://domain/abc: EOF"},
{"http://domain/abc", "get(read) from", io.EOF,
	 false, false, "get(read) from http://domain/abc: EOF",
},
{"http://domain/abc", "post(write) to", fakeOpError{true, false},
	true, false, "post(write) to http://domain/abc: fake error",
},
{"http://domain/abc", "get(read) from", fakeOpError{false, true},
	false, true, "get(read) from http://domain/abc: fake error",
}
```

Something like that. All cases wrap at `timeout` field.

Reply:
fixed